docs/design-documents/features/storage/Configuration/CONFIGURATION.md

```diff
@@ -62,6 +62,8 @@ The following is a list of all storage parameters available and their descriptio
     * `TDB_EXTERNAL_NO_RBP`.
     * `FILESYSTEM`.
     * `FILESYSTEM_NO_RBP`.
+    * `default`
+  If default is set, the system will choose the type of storage base on the block device component set in target.json. For QSPIF, SPIF and DATAFLASH block devices TDB_EXTERNAL will be used. If SD is set a FILESYSTEM storage is set and if only FLASHIAP exists as the only block device component, a TDB_INTERNAL will be used.  
```

Comment:
@AnotherButler this addition to docs is confusing, could you help us reword it a bit to make it clearer?

Reply:
As Amanda is not available this week, can this be addressed separately (@yossi2le can you make sure it will be updated )

Reply:
No problem, I will take for that next week,